MaterialLib/SolidModels/Ehlers-impl.h

```diff
@@ -702,31 +742,25 @@ SolidEhlers<DisplacementDim>::integrateStress(
         dresidual_deps.template block<KelvinVectorSize, KelvinVectorSize>(0, 0)
             .noalias() = calculateDResidualDEps<DisplacementDim>(mp.K, mp.G);
 
-        if (_damage_properties)
-            updateDamage(state.eps_p_V - state.eps_p_V_prev,
-                         state.eps_p_eff - state.eps_p_eff_prev, state.damage,
-                         t, x, state);
-
-        // Extract consistent tangent.
         *tangentStiffness =
             mp.G *
             linear_solver.solve(-dresidual_deps)
                 .template block<KelvinVectorSize, KelvinVectorSize>(0, 0);
     }
 
     if (_damage_properties)
-        return {mp.G * sigma * (1 - state.damage.damage),
+        return std::make_tuple(mp.G * sigma * (1 - state.damage.value()),
```

Comment:
Checking this out one more time, but if I remember correctly there was a compiler complaining about braces.